std_msgs/Header header
string[] name
float32[] position
---
bool[] at_tgt
